Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -346,13 +294,31 @@ public struct ConvertAction: AsyncAction {
             indexer: indexer,
             enableCustomTemplates: experimentalEnableCustomTemplates,
             transformForStaticHostingIndexHTML: transformForStaticHosting ? indexHTML : nil,
-            bundleIdentifier: bundleIdentifier
+            bundleIdentifier: bundle.identifier
         )
 
+        if experimentalModifyCatalogWithGeneratedCuration, let catalogURL = rootURL {
+            let writer = GeneratedCurationWriter(context: context, catalogURL: catalogURL, outputURL: catalogURL)
+            let curation = try writer.generateDefaultCurationContents()
+            for (url, updatedContent) in curation {
+                guard let data = updatedContent.data(using: .utf8) else { continue }
+                try? FileManager.default.createDirectory(at: url.deletingLastPathComponent(), withIntermediateDirectories: true, attributes: nil)
+                try? data.write(to: url, options: .atomic)
+            }
+        }
+        
         let analysisProblems: [Problem]
         let conversionProblems: [Problem]
         do {
-            (analysisProblems, conversionProblems) = try converter.convert(outputConsumer: outputConsumer)
+            conversionProblems = try ConvertActionConverter.convert(
```

Comment:
I agree that it's very repetitive but the one thing I like abut that name is that it signals that the API is currently only intended for one caller. 

If it had a more general name then it would invite generalization that I don't think we have enough information about use-cases to do well. 

The suggestion about `DocumentationContextConverter` is interesting but mostly because I don't see a strong reason for that type to exist in its current form.